xml/System.Windows.Forms/NotifyIcon.xml

```diff
@@ -1110,7 +1110,7 @@
   
  ]]></format>
         </remarks>
-        <exception cref="T:System.ArgumentException">ToolTip text is more than 63 characters long.</exception>
+        <exception cref="T:System.ArgumentException">ToolTip text is more than 127 characters long (.NET 6 and later only) or more than 63 characters long (.NET Framework, .NET Core, and .NET 5 only).</exception>
```

Comment:
Not long ago in https://github.com/dotnet/dotnet-api-docs/pull/5096 we made a similar update, but it was done in a tabular/list form. Do you think we can do a similar thing here?

Reply:
How does this look? https://review.docs.microsoft.com/en-us/dotnet/api/system.windows.forms.notifyicon.text?view=net-5.0&branch=pr-en-us-5258#exceptions

Reply:
I like that! 😀